Update insert-functions.md - insert funk FAQ logging limit (#6761)
segmentio/segment-docs: src/connections/functions/insert-functions.md

For more information, visit Segment's [Public API docs](https://docs.segmentapis.com/tag/Functions){:target="_blank"}.

{% endcomment %}

##### What is the maximum data size that can be displayed in console.logs() when testing a Function?

> The test function interface has a 4KB console logging limit. Outputs surpassing this limit will not be visible in the user interface.

Reviewer: ```suggestion
The test function interface has a 4KB console logging limit. Outputs surpassing this limit won't be visible in the user interface.
```
